build.sbt

```
@@ -6,19 +6,22 @@ ThisBuild / developers ++= List(
   tlGitHubDev("archisman-dey", "Archisman Dey"),
   tlGitHubDev("armanbilge", "Arman Bilge"),
   tlGitHubDev("valencik", "Andrew Valencik"),
+  tlGitHubDev("pantShrey", "Shrey Pant"),
```

You are lead on this project now :)

```suggestion
  tlGitHubDev("pantShrey", "Shrey Pant"),
  tlGitHubDev("armanbilge", "Arman Bilge"),
  tlGitHubDev("valencik", "Andrew Valencik"),
```